src/components/DocumentationTopic.vue

```diff
@@ -97,7 +94,6 @@ import Relationships from './DocumentationTopic/Relationships.vue';
 import RequirementMetadata from './DocumentationTopic/Description/RequirementMetadata.vue';
 import Availability from './DocumentationTopic/Summary/Availability.vue';
 import SeeAlso from './DocumentationTopic/SeeAlso.vue';
-import Summary from './DocumentationTopic/Summary.vue';
```

Comment:
Yes, I will remove this component.
In fact, since this component is no longer being used, we should re-organize the folder too. But I will do that in another PR, since it shouldn't block us from merging this.